src/olympia/search/filters.py

```diff
@@ -382,37 +382,32 @@ def secondary_should_rules(self, search_query, analyzer):
         """Return "secondary" should rules for the query.
 
         These are the ones using the weakest boosts, they are applied to fields
-        containing more text like description, summary and tags.
+        containing more text like description & summary.
 
         Applied rules:
 
-        * Look for phrase matches inside the summary (boost=0.8)
+        * Look for phrase matches inside the summary (boost=2.0)
         * Look for phrase matches inside the summary using language specific
-          analyzer (boost=0.6)
-        * Look for phrase matches inside the description (boost=0.3).
+          analyzer (boost=3.0)
+        * Look for phrase matches inside the description (boost=2.0).
         * Look for phrase matches inside the description using language
-          specific analyzer (boost=0.1).
-        * Look for matches inside tags (boost=0.1).
+          specific analyzer (boost=3.0).
         """
         should = [
-            query.MatchPhrase(summary={'query': search_query, 'boost': 0.8}),
+            query.MatchPhrase(summary={'query': search_query, 'boost': 2.0}),
             query.MatchPhrase(description={
-                'query': search_query, 'boost': 0.3}),
+                'query': search_query, 'boost': 2.0}),
         ]
 
-        # Append a separate 'match' query for every word to boost tag matches
-        for tag in search_query.split():
-            should.append(query.Match(tags={'query': tag, 'boost': 0.1}))
```

Comment:
Removed as tags are not exposed on the frontend and review page, it's to easy to insert a bunch of tags to increase ranking.

Reply:
this would have _decreased_ their ranking instead, right?

Reply:
What I meant in my comment is: if I had fixed the boost to increase the score instead of decrease it, as it was meant to be, it would be easy to game.